### DISCUSSION TITLE
How to use the 'util' package to allow all modules to depend on it?

### DISCUSSION THREAD

**Discussion by majian159** [spring-projects/spring-modulith#352] [Q&A]
```
- infrastructure
  |-- exception
  |-- logging
  |-- xxx

- module1
- module2

```

How can I allow module1 and module2 to directly use types from the subpackages under 'infrastructure'?

**Comment by nkolosnjaji** [comment]
You can try with `@Modulithic(sharedModules = "infrastructure")` with your `@SpringBootApplication`

**Reply by bl4keee** [reply]
That does not seem to work.

**Comment by bl4keee** [comment]
Hi, did you manage to find a solution? @majian159

**Reply by cjacky475** [reply]
Is this not yet available @bl4keee, @majian159 or did you guys found out how to do this?

**Comment by nkolosnjaji** [comment]
then use named interfaces

https://docs.spring.io/spring-modulith/reference/1.1/fundamentals.html#modules.named-interfaces
